library/std/src/panicking.rs

```diff
@@ -453,7 +453,9 @@ pub fn begin_panic<M: Any + Send>(msg: M) -> ! {
         intrinsics::abort()
     }
 
-    rust_panic_with_hook(&mut PanicPayload::new(msg), None, Location::caller());
+    return crate::sys_common::backtrace::__rust_end_short_backtrace(move || {
```

Comment:
Could you also do this in `begin_panic_handler`?

Reply:
Good spot.  Added for next push.